cmd/peapod-to-fstree/main.go

```diff
@@ -110,12 +139,14 @@ func main() {
 
 		log.Printf("migrating data from Peapod '%s' to Fstree '%s' (batches of %d objects)...\n", ppd.Path(), fstr.Path(), countLimit)
 
-		err = common.CopyBatched(fstr, ppd, countLimit)
+		onlyActualObjectsPeapod := storageNoRottenObjects{Storage: ppd, actualContainers: cnrs}
+
+		err = common.CopyBatched(fstr, &onlyActualObjectsPeapod, countLimit)
 		if err != nil {
 			log.Fatal("migration failed: ", err)
 		}
 
-		log.Printf("data successfully migrated in the shard %d, going to the next one...\n", i)
+		log.Printf("data successfully migrated (read objects: %d, skipped: %d) in the shard %d, going to the next one...\n", i, onlyActualObjectsPeapod.read, onlyActualObjectsPeapod.skipped)
```

Comment:
```suggestion
		log.Printf("data successfully migrated (read objects: %d, skipped: %d) in the shard %d, going to the next one...\n", onlyActualObjectsPeapod.read, onlyActualObjectsPeapod.skipped, i)
```
or move the brackets

Reply:
god bless review